### PR TITLE
Siril, new release update

### PR DESCRIPTION
**Pull request by lock042**
Update

### CODE REVIEW

File: siril.rb [hunk 1]
```diff
@@ -1,18 +1,10 @@
...
   desc "Astronomical image (pre-)processing application"
   homepage "http://free-astro.org/index.php/Siril"
-  url "https://free-astro.org/download/siril-0.9.5.tar.bz2"
-  sha256 "8f25a8cb8dc1f2ca9da979161a51e6aacd4059674e21ee14edcc0f299e2a7924"
-  revision 1
+  url "https://free-astro.org/download/siril-0.9.6.tar.bz2"
+  sha256 "0622c885667396f1181f5c727ac1f8e3985560de817fec4400cd7d5ae6ed970f"
   head "https://free-astro.org/svn/siril/"
 
-  bottle do
```
Comment (iMichka) [inline siril.rb]: Don't remove the bottle block. We will update this for you when merging the pull request.

You can make the change, then `git add .`, `git commit --amend` to edit your commit, and `git push --force` to your fork to update the pull request.

Comment (lock042) [inline siril.rb]: I've done this. Is this ok ?

File: siril.rb [hunk 2]
```diff
@@ -1,9 +1,8 @@
-class Siril < Formula
```
Comment (iMichka) [inline siril.rb]: Don't remove the class here.

Comment (lock042) [inline siril.rb]: don't know why I've done this ... Sorry.

Comment (lock042) [inline siril.rb]: It's ok.
The formula should be fine now.

**Comment by lock042** [comment]
@iMichka : Is everything ok for an update now ?

**Comment by iMichka** [comment]
@lock042 Thanks for your contribution! We can ignore the Linux builds for the moment as there were no Linux bottles here before.

**Pull request closed without merging**